tests/e2e/lib.sh

```diff
@@ -276,7 +273,17 @@ deploy_sensor_via_operator() {
         --output-secrets - \
     | kubectl -n stackrox apply -f -
 
-    kubectl apply -n stackrox -f tests/e2e/yaml/secured-cluster-cr.yaml
+    if [[ -n "${COLLECTION_METHOD:-}" ]]; then
+	echo "Overriding the product default collection method due to COLLECTION_METHOD variable: ${COLLECTION_METHOD}"
```

Comment:
Tabs/spaces indentation looks not matching. Here it should be:

```suggestion
        echo "Overriding the product default collection method due to COLLECTION_METHOD variable: ${COLLECTION_METHOD}"
```

Reply:
Done